Backpack-Compose/build.gradle

```diff
@@ -0,0 +1,91 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+
+version = rootProject.ext.version
+group = 'net.skyscanner.backpack-compose'
+
+repositories {
+  google()
+  mavenCentral()
+}
+android {
```

Comment:
there seems to be quite a lot of duplication - can the generic bits be moved into a separate gradle file that gets applied?